model-engine/model_engine_server/inference/infra/gateways/firehose_streaming_storage_gateway.py

```diff
@@ -29,7 +29,8 @@ def __init__(self):
     """
 
     def _get_firehose_client(self):
-        sts_client = boto3.client("sts", region_name=infra_config().default_region)
+        sts_session = boto3.Session(region_name=infra_config().default_region)
```

Comment:
possible to create this client at the process level?

Reply:
sorry what do you mean by "process level"?

Reply:
like using a global variable

Reply:
I don't reuse it anywhere else actually so doesn't need to be global

also found this lol: https://stackoverflow.com/questions/19158339/why-are-global-variables-evil